data/templates/template.typst

```diff
@@ -16,36 +16,62 @@
   keywords: (),
   date: none,
   abstract: none,
+  thanks: none,
   cols: 1,
   margin: (x: 1.25in, y: 1.25in),
   paper: "us-letter",
   lang: "en",
   region: "US",
   font: (),
   fontsize: 11pt,
+  mathfont: "New Computer Modern Math",
+  codefont: "DejaVu Sans Mono",
+  linestretch: 1,
   sectionnumbering: none,
+  linkcolor: ["#800000"],
+  citecolor: ["#0000FF"],
+  toccolor: ["#800000"],
```

Comment:
Again, is this the same as Typst's default behavior?  If not, why these colors?  Would it perhaps be better to make Typst use its standard defaults unless these are specified?